### ISSUE TITLE
Training Error!!

### ISSUE THREAD

**Issue by aybukesakaci** [RicardooYoung/VersatileTeacher#1]
Hello,

i got an error while the training.

While executing this line:" !python train.py --cfg models/yolov5l_da.yaml --weights weights/yolov5l.pt --data data/src_data.yaml --data data/tgt_data.yaml --device 0 --epochs 50 --batch-size 16" 

Getting this error:
![image](https://github.com/RicardooYoung/VersatileTeacher/assets/10299065/f76e927e-2a20-4522-9acf-f9db41a1339c)

Thanks in advance!

**Comment by aybukesakaci** [comment]
And also could you to upload the parameters you use? such as src_data.yaml, tgt_data.yaml or hyp.scratch-low.yaml etc. This way, I can easily compare with my own case.
Thanks!

**Comment by RicardooYoung** [comment]
Hello, @aybukesakaci.

There is a bug in the code where mask should be set to empty list if only one model is employed. Fixed.

The parameters I use are all in the folder _data_, including hyperparameters and dataset configs. All config files stay same with the original YOLO.

Best regards.

**Comment by aybukesakaci** [comment]
Thank you for your quick return. I ran the code as explained. It worked. My question to you is this: When I want to use val.py or detect.py, I get an error like this. 

![image](https://github.com/RicardooYoung/VersatileTeacher/assets/10299065/1735bac2-5062-46e5-9a36-423e83d1e6d6)


How can I use it when I want to use the test set without getting an error?

Thanks in advance!

**Comment by RicardooYoung** [comment]
Hello @aybukesakaci 

The missing files have been uploaded.

Best regards.
